### PR TITLE
fix a bug about timezone

### PR DESCRIPTION
**Pull request by iorikingdom**

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by ericprieto** [comment]
This solved my problem, I'd like to have it merged. Can I help in some way?

**Comment by raymondfeng** [comment]
@iorikingdom Thank you for the patch. 

1. Would you please sign CLA - https://cla.strongloop.com/agreements/strongloop/loopback-connector-mysql?

2. Please also provide some context on what problem your patch tries to solve? We had similar discussions on TimeZone for Postgresql. I would like to make sure they are consistent.

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/strongloop/loopback-connector-mysql.git"
+    "url": "https://github.com/iorikingdom/loopback-connector-mysql.git"
```
Comment (raymondfeng) [inline package.json]: Please remove this change from the PR.

**Comment by ericprieto** [comment]
Any news about this? I'm still using @iorikingdom repo for my loopback projects

**Comment by darknos** [comment]
I propose another fix of the same problem:
https://github.com/strongloop/loopback-connector-mysql/pull/170

1. it handles dateStrings connection params
2. pass date fields as is to node-mysql (without UTC converting)
3. fix issue with '0000-00-00' dates

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: lib/mysql.js [hunk 2]
```diff
@@ -351,7 +351,7 @@ MySQL.prototype.fromColumnValue = function(prop, val) {
         val = String(val);
         break;
       case 'Date':
-        val = new Date(val.toString().replace(/GMT.*$/, 'GMT'));
+        //val = new Date(val.toString().replace(/GMT.*$/, 'GMT'));
```
Comment (b-admike) [inline lib/mysql.js]: If this line is no longer needed, then you can remove it instead of commenting it.

**Comment by b-admike** [comment]
@iorikingdom thank you for your contribution. It seems like #170 also provides the changes that are presented in this PR and this PR has been stale for a year and a half. I will close it now, but we always welcome your contribution. @ericprieto please have a look at @darknos's solution in #710.

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**